pages/portfolio.html

```diff
@@ -5,8 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Cormorant">
 </head>
 <body>
-  
+  <header>
+    <h1 class="site-title">M Chau</h1>
```

Comment:
While using a class here is fine, I think since this site title only shows up once per page (or at least it _should_ only show up once per page), it can probably be an ID instead.